<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>testVSClass</class>
 <widget class="QWidget" name="testVSClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>testVS</string>
  </property>
  <widget class="QPushButton" name="Wroclaw_Btn">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>380</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Wr</string>
   </property>
  </widget>
  <widget class="QLabel" name="miasto_p">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>90</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Wybierz miasto</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>631</width>
     <height>601</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Poznan_Btn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>260</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Po</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Szczecin_Btn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Sz</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ZielonaG_Btn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Z.G</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Gdansk_Btn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Gd</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Warszawa_Btn">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>250</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Wwa</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Rzeszow_Btn">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>470</y>
     <width>21</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Rz</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Krakow_Btn">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>490</y>
     <width>21</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Kr</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Lublin_Btn">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>370</y>
     <width>21</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Lu</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Bialystok_Btn">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>130</y>
     <width>21</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Bi</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Olsztyn_Btn">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>110</y>
     <width>21</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Ol</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Katowice_Btn">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>440</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Kat</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Kielce_Btn">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>400</y>
     <width>21</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Ki</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Bydgoszcz_Btn">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>170</y>
     <width>21</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>By</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Lodz_Btn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>330</y>
     <width>21</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Łó</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>60</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Miasto początkowe:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>120</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Miasto końcowe:</string>
   </property>
  </widget>
  <widget class="QLabel" name="miasto_k">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>150</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Wybierz miasto</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>190</y>
     <width>128</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="radioButton">
      <property name="text">
       <string>Miasto początkowe</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="text">
       <string>Miasto końcowe</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="Dij_Btn">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>280</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Oblicz droge</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>320</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Najkrótsza droga:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Dij_label">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>360</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Opole_Btn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>410</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Op</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>250</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>246</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>190</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>200</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>110</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>223</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>430</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>184</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>310</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>173</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>380</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>195</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>490</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>172</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>440</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>155</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>450</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>114</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>480</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>80</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>330</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>181</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>290</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>142</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>200</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>214</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>140</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>224</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>250</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>226</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>110</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>221</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>420</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>156</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>390</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>203</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>430</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>104</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>410</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>97</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>330</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>182</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_26">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>220</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>138</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_27">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>370</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>153</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_28">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>265</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_29">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>213</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_30">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>330</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>166</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_31">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>358</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_32">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>70</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>166</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_33">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>153</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_34">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>310</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>206</string>
   </property>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>miasto_p</zorder>
  <zorder>graphicsView</zorder>
  <zorder>Wroclaw_Btn</zorder>
  <zorder>Poznan_Btn</zorder>
  <zorder>Szczecin_Btn</zorder>
  <zorder>ZielonaG_Btn</zorder>
  <zorder>Gdansk_Btn</zorder>
  <zorder>Warszawa_Btn</zorder>
  <zorder>Rzeszow_Btn</zorder>
  <zorder>Krakow_Btn</zorder>
  <zorder>Lublin_Btn</zorder>
  <zorder>Bialystok_Btn</zorder>
  <zorder>Olsztyn_Btn</zorder>
  <zorder>Katowice_Btn</zorder>
  <zorder>Kielce_Btn</zorder>
  <zorder>Bydgoszcz_Btn</zorder>
  <zorder>Lodz_Btn</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>miasto_k</zorder>
  <zorder>Dij_Btn</zorder>
  <zorder>label</zorder>
  <zorder>Dij_label</zorder>
  <zorder>Opole_Btn</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>label_9</zorder>
  <zorder>label_10</zorder>
  <zorder>label_11</zorder>
  <zorder>label_12</zorder>
  <zorder>label_13</zorder>
  <zorder>label_14</zorder>
  <zorder>label_15</zorder>
  <zorder>label_16</zorder>
  <zorder>label_17</zorder>
  <zorder>label_18</zorder>
  <zorder>label_19</zorder>
  <zorder>label_20</zorder>
  <zorder>label_21</zorder>
  <zorder>label_22</zorder>
  <zorder>label_23</zorder>
  <zorder>label_24</zorder>
  <zorder>label_25</zorder>
  <zorder>label_26</zorder>
  <zorder>label_27</zorder>
  <zorder>label_28</zorder>
  <zorder>label_29</zorder>
  <zorder>label_30</zorder>
  <zorder>label_31</zorder>
  <zorder>label_32</zorder>
  <zorder>label_33</zorder>
  <zorder>label_34</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="testVS.qrc"/>
 </resources>
 <connections/>
</ui>
